docs/extras/xdebug.md

```diff
@@ -14,6 +14,11 @@ To use XDebug with PHPStorm, change the `php_xdebug_idekey` variable as shown be
 ```yaml
 php_xdebug_idekey: PHPSTORM
 ```
+In order to pass the Web Server Debug Validation, you'll need to also add the following your `config.yml`
+
+```yaml
+php_xdebug_remote_host: "{{ vagrant_hostname }}"
```

Comment:
@geerlingguy correct, AFAIK there's no stable way to get the IP/hostname of the host (machine running the VM) from _inside_ the VM.

It looks like [vagrant-hostmanager](https://github.com/devopsgroup-io/vagrant-hostmanager) also manages the hosts file of the guest (VM).